Sources/PayPalMessages/Enums/Environment.swift

```diff
@@ -78,6 +78,24 @@ public enum Environment: Equatable {
         case log = "/v1/credit/upstream-messaging-events"
     }
 
+    public var devTouchpoint: Bool {
+        switch self {
+        case .local(_, let devTouchpoint, _), .stage(_, let devTouchpoint, _):
+            return devTouchpoint
+        default:
+            return false
+        }
+    }
+
+    public var stageTag: String? {
+        switch self {
+        case .local(_, _, let stageTag), .stage(_, _, let stageTag):
+            return stageTag
+        default:
+            return nil
+        }
+    }
```

Comment:
Instead of exposing a `devToucpoint` and `stageTag` field on the environment that replaces `config.data.devTouchpoint` with `environment.devTouchpoint`, could we instead leverage the values internally within the `Environment` enum to auto include them as part of the `url()` method? They are ultimately only ever used as extra query params so could we not keep the logic encapsulated within the `Environment` to simplify any other parts of the code being responsible for it and needing to drill it down to where it needs to be?